### PR TITLE
examples form onsubmit

### PR DESCRIPTION
**Pull request by tonyspiro**

### CODE REVIEW

File: examples/todos-with-undo/containers/AddTodo.js [hunk 1]
```diff
@@ -7,15 +7,19 @@ let AddTodo = ({ dispatch }) => {
...
   return (
     <div>
-      <input ref={node => {
-        input = node
-      }} />
-      <button onClick={() => {
-        dispatch(addTodo(input.value))
+      <form onSubmit={e => {
+        e.preventDefault()
+        if(input.value.trim())
```
Comment (sompylasar) [inline examples/todos-with-undo/containers/AddTodo.js]: Coding style:
```js
        if (input.value.trim()) {
          dispatch(addTodo(input.value))
        }
```

File: examples/todos/containers/AddTodo.js [hunk 2]
```diff
@@ -7,15 +7,19 @@ let AddTodo = ({ dispatch }) => {
...
   return (
     <div>
-      <input ref={node => {
-        input = node
-      }} />
-      <button onClick={() => {
-        dispatch(addTodo(input.value))
+      <form onSubmit={e => {
+        e.preventDefault()
+        if(input.value.trim())
```
Comment (sompylasar) [inline examples/todos/containers/AddTodo.js]: Coding style:
```js
        if (input.value.trim()) {
          dispatch(addTodo(input.value))
        }
```

**Comment by gaearon** [comment]
What is the benefit of this? Does this make Enter work?

**Comment by tonyspiro** [comment]
Yes it makes enter work as well as the submit button click.

File: examples/todos-with-undo/containers/AddTodo.js [hunk 3]
```diff
@@ -7,15 +7,20 @@ let AddTodo = ({ dispatch }) => {
...
   return (
     <div>
-      <input ref={node => {
-        input = node
-      }} />
-      <button onClick={() => {
-        dispatch(addTodo(input.value))
+      <form onSubmit={e => {
+        e.preventDefault()
+        if (input.value.trim()) {
```
Comment (gaearon) [inline examples/todos-with-undo/containers/AddTodo.js]: Let's make this an early return instead.

**Comment by gaearon** [comment]
Can you please apply the same change to all other examples in docs and `examples` folder? I think these lines should have a couple more matches across the codebase.

**Comment by tonyspiro** [comment]
@gaearon will do :+1:

**Comment by tonyspiro** [comment]
@gaearon this should be good to go.  Please let me know if I missed anything.

**Comment by gaearon** [comment]
Thank you!

**Comment by tonyspiro** [comment]
np :smile: thank you!

**Pull request merged**